### DISCUSSION TITLE
[FR]:  问个问题 为什么taro.tsx文件里面 有时候div不用view

### DISCUSSION THREAD

**Discussion by adolfheir** [jdf2e/nutui-react#2065] [Q&A] [answered]
### 这个功能解决了什么问题？

问个问题 为什么taro.tsx文件里面 有时候div不用view
比如这个地方  是因为没适配完 还是有什么故意的逻辑？
https://github.com/jdf2e/nutui-react/blob/next/src/packages/pulltorefresh/pulltorefresh.taro.tsx#L170

### 你期望的组件设计是怎样的？

希望在文档里写一下

**Comment by eiinu** [comment, marked as answer]
在使用 [@tarojs/plugin-html](https://taro-docs.jd.com/docs/use-h5) 插件的情况下，div 标签会被映射为 View 组件，二者没有区别。
